app/controllers/users_controller.rb

```diff
@@ -0,0 +1,40 @@
+class UsersController < ApplicationController
+  include UsersHelper
+  def new
+    @user = User.new
+  end
+
+  def index
+    @users = User.all
+  end
+
+  def show
+    @user = User.find(params[:id])
+  end
```

Comment:
These two actions will be redundant when you use the` :only` option in your `config/routes.rb`.